webapp-django/crashstats/topcrashers/views.py

```diff
@@ -0,0 +1,302 @@
+import datetime
+import isodate
+from collections import defaultdict
+
+from django import http
+from django.conf import settings
+from django.shortcuts import render, redirect
+from django.utils import timezone
+from django.utils.http import urlquote
+
+from waffle.decorators import waffle_switch
+
+from session_csrf import anonymous_csrf
+
+from crashstats.crashstats import models
+from crashstats.crashstats.decorators import (
+    check_days_parameter,
+    pass_default_context,
+)
+from crashstats.supersearch.form_fields import split_on_operator
+from crashstats.supersearch.models import (
+    SuperSearchUnredacted,
+)
+
+
+def get_date_boundaries(parameters):
+    """Return the date boundaries in a set of parameters.
+
+    Return a tuple with 2 datetime objects, the first one is the lower bound
+    date and the second one is the upper bound date.
+    """
+    default_date_range = datetime.timedelta(days=7)
+
+    greater_than = None
+    lower_than = None
+
+    if not parameters.get('date'):
+        lower_than = timezone.now()
+        greater_than = lower_than - default_date_range
+    else:
+        for param in parameters['date']:
```

Comment:
No, that's just the safest way to process dates from a Super Search form. Even if it isn't used now, I'd like to keep it around so that it becomes very simple to build a more "Super Search-y" interface on top of this.